### PR TITLE
#4322 [Manager] Refactor nodes API with Redux

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4322 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by jackyoh** [comment]
漏指定到 Assignees 了

File: ohara-manager/client/src/store/epics/appEpics.js [hunk 1]
```diff
@@ -30,15 +30,18 @@ const initializeAppEpic = action$ =>
       zip(
         apiWrapper.wrapFetchWorkspaces(),
         apiWrapper.wrapFetchPipelines(),
+        apiWrapper.wrapFetchNodes(),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/appEpics.js]: 我們應該盡可能降低 App initialization 的時間。Nodes 是否可以在頁面需要的時候再打 API ?

File: ohara-manager/client/src/store/epics/nodeEpics.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        map(entities => actions.fetchNodes.success(entities)),
+        startWith(actions.fetchNodes.request()),
+        catchError(res => of(actions.fetchNodes.failure(res))),
+      ),
+    ),
+  );
+
+let retryCount = 0;
+
+const fetchNode = values =>
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/nodeEpics.js]: 我們可能需要對 RxJS Observable 規範一個 naming rule，這樣會比較容易瞭解該用途。

我個人頃向使用 $ 作為結尾，例如 fetchNode$

File: ohara-manager/client/src/store/epics/nodeEpics.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    map(action => action.payload),
+    switchMap(values =>
+      of(
+        from(apiWrapper.wrapDeleteNode({ hostname: values })).pipe(
+          map(() => actions.deleteNode.request()),
+          startWith(actions.deleteNode.request()),
+          catchError(res => of(actions.deleteNode.failure(res))),
+        ),
+        fetchNode(values),
+      ).pipe(concatAll()),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/nodeEpics.js]: Good!

File: ohara-manager/client/src/store/epics/nodeEpics.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/store/epics/nodeEpics.js]: ok thanks

File: ohara-manager/client/src/store/epics/appEpics.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/store/epics/appEpics.js]: ok i will change

**Pull request merged**